src/components/Auth/AuthMainForm.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import styled from 'styled-components';
+import Switch from 'components/Shared/Switch';
+import StyledButton from 'components/Shared/Button';
+import PwdInput from 'components/Auth/PwdInput';
+import * as S from 'components/Auth/styles';
+
+const AuthForm = styled.div``;
+
+const AuthMainForm = () => (
+  <AuthForm>
+    <S.StyledInput autocomplete="username" name="username" placeholder="아이디 입력" />
+    <PwdInput placeholder="비밀번호 입력" />
+    <S.AutoLogin>
+      <S.AutoLoginCheck>
+        <Switch />
+      </S.AutoLoginCheck>
+      <S.AutoLoginText>자동 로그인</S.AutoLoginText>
+    </S.AutoLogin>
+    <StyledButton>로그인</StyledButton>
+    <S.OtherOption>
+      <S.StyledLink to="idfind">아이디 찾기</S.StyledLink>
+      <p> | </p>
+      <S.StyledLink to="pwdfind">비밀번호 찾기</S.StyledLink>
+      <div>|</div>
+      <S.StyledLink to="register">회원가입</S.StyledLink>
+    </S.OtherOption>
+    <S.SnsLoginText>SNS로 간편 로그인하기</S.SnsLoginText>
```

Comment:
`acronym` 은 소문자보단 대문자를 기입하는게 좋습니다.

```suggestion
    <S.SNSLoginText>SNS로 간편 로그인하기</S.SNSLoginText>
```

Reply:
엌 왜 만들땐 저런 이상함을 눈치 못챌까요

Reply:
725915c